.github/workflows/build.yml

```
@@ -95,6 +95,8 @@ jobs:
           - {goos: "linux", goarch: "amd64"}
           - {goos: "linux", goarch: "arm"}
           - {goos: "linux", goarch: "arm64"}
+          - {goos: "darwin", goarch: "arm"}
```

note, this is wrong:
```suggestion
          - {goos: "darwin", goarch: "amd64"}
```